### DISCUSSION TITLE
Vente d'objet

### DISCUSSION THREAD

**Discussion by RayZZo9** [Crownicles/Crownicles#1902] [Suggestions fermées]
On peut déjà vendre des objets, mais je me demandais si c'était possible de faire que l'on puisse les vendre à un joueur, de la même façon qu'un familier. Cela permettrais de faire du commerce et de pouvoir choisir soi-même le prix de son objet.
Ce serai une nouvelle commande ou un paramètre à mettre dans la commande /vendre

Qu'en pensez-vous ?

**Comment by voltou91** [comment]
Salut, moi je suis contre pour une raison.
Cela encouragerait le feed par double compte.
Exemple tu débute l'aventure et tu te créé plusieurs compte pour revendre à 1€ ton item a ton compte principal.

**Comment by RayZZo9** [comment]
Awi
c'est vrai ça, j'y avais pas pensé.

On pourrait demander aux gens de ne pas avoir de deuxième compte...
Mais bon...
ouaip

**Reply by Easygamer01s** [reply]
Bonjour,

J'y ai déjà réfléchi aussi.. la mécanique est dangereuse pour le multi compte car on peut choisir la contrepartie..

On pourrait imaginer qu'il serait possible de faire concurrence aux vendeurs que l'on croise dans les mini events, donc pas le choix de qui va acheter..

Exemple :
J'ai un bel idem dans mon sac à dos.
Avec la commande /sell je le mets en vente, je détermine un prix..
Le prix, l'objet et mon identifiant est stocké dans une bdd. 
Quand un autre joueur fait /report, il tombe aléatoirement sur un vendeur avec ma proposition. 
Il l'accepte, je reçois l'argent lors de mon prochain /report..


Bien-sûr, il faudrait limiter la durée de vente, et le nombre d'items pour que la bdd soit pas folle..

**Reply by romain22222** [reply]
tiens ca recoupe avec une autre idée que j'avais eu pour... un truc qui vient, mais moi je suis hyper pour.

Je rajouterais juste une sorte d'expiration d'item, qui, s'il n'est pas récupéré dans la semaine, alors il est détruit

**Reply by romain22222** [reply]
autre chose, il faudra que la vente soit anonyme, afin d'éviter un self feed bizarre

**Reply by BastLast** [reply]
Ya deja un mini event prévu pour acheter des légendaire et épiques

**Reply by romain22222** [reply]
draft c'est pas juste un mini event shop
c'est un mini event de marché gris, en p2p

**Reply by romain22222** [reply]
mais p2p à double aveugle

**Reply by RayZZo9** [reply]
C'est une bonne idée, une bonne variation...

Par contre si tout le monde fait la commande sell il y aura beaucoup trop d'objet...
Mais, ducoup on pourrait faire que, il y a un mini-event où on croise un vendeur itinérant, il nous propose de nous acheter un objet, et il nous demande le prix (et l'objet), le prix ainsi que l'objet sont stoqués dans une liste, et quand quelqu'un tombe sur un mini-event avec un marchand c'est un objet de cette liste qui est mis. Le vendeur initial (le joueur) reçoit l'argent (ou la moitié) à son prochain mini-event grâce à ce bon vieux marchand qui est allé vendre nos affaires à notre place.

**Comment by Easygamer01s** [comment]
Je trouverais ça cool de voir passer des légendaires se vendre 100k dans le mini event, qu'il s'agisse du drop d'un autre joueur..
